apps/server/priv/boot.exs

```diff
@@ -0,0 +1,17 @@
+[script_dir] = System.argv()
+
+[script_dir, "..", "lib", "*.ez"]
+|> Path.join()
+|> Path.wildcard()
+|> Enum.each(fn archive_path ->
```

Comment:
i _think_ this can be accomplished by 
```elixir
[System.cwd!(), "lib", "*.ez"]
|> Path.join()
|> Path.wildcard()
|> Enum.map(&Path.join(&1, "ebin"))
|> :code.add_patha()
```

Reply:
This will require adding the bin directory via `-pa`

Reply:
This doesn't work because we need the name of the lib betwen `.ez` and `ebin`